src/main/resources/config.yml

```diff
@@ -120,6 +120,7 @@ blocked_commands:
   - 'n:b:/setjail:_'
   - 'n:b:/tfbanlist:Use /banlist to see and wipe the ban list'
   - 'n:b:/tfipbanlist:Use /banlist to see and wipe the ban list'
+  - 'n:b:/summon:This command has been blocked for security reasons'
```

Comment:
I think this should just be an empty '_' message. Since most of these commands are blocked for "security measures". The empty message implies this.